cloudflareupdater.py

```diff
@@ -0,0 +1,206 @@
+import boto3
```

Comment:
Boo for using python instead of node.

Reply:
I suck at node more than I suck at pretty much anything else.

Reply:
You don't need promises in a synchronous world.

Reply:
No, just a functional language and much better json support

Reply:
I think I'd like node better if not for the horribly complicated ways people tend to do otherwise simple things in node.

Reply:
Once I got used to functional programming, OO with Python or anything else started seeming horribly complicated to me

Reply:
I want to learn Haskell

Reply:
I don't mind the functional aspects of node, it's the asynchronous nature of it that annoys the crap out of me.

Reply:
I think async is the future. You can't be waiting on callbacks when everything is a function accessed via http